CONTRIBUTING.md

```diff
@@ -220,16 +216,17 @@ Run the tests
 
 Three ways of running the tests are as follows:
 
-1.  `make test` (which uses `docker`)
 2.  `./run-tests.sh` or `run-tests.bat`
+  - NOTE: `./run-tests.sh` is not working right now due to [this issue](https://github.com/pypa/pip/issues/7953#issuecomment-611928262).
 3.  Using pipenv:
 
 ``` {.console}
 $ git clone https://github.com/pypa/pipenv.git
 $ cd pipenv
 $ git submodule sync && git submodule update --init --recursive
 $ pipenv install --dev
-$ pipenv run pytest
+# CI=1 is necessary as a workaround for https://github.com/pypa/pipenv/issues/4909
```

Comment:
Would be better to set the env in `tests/integration/conftest.py`?

Reply:
Oh that's brilliant. Changed!